lib/parachain/statement_fetching.go

```diff
@@ -0,0 +1,81 @@
+package parachain
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+// Request for fetching a large statement via request/response.
+type StatementFetchingRequest struct {
+	// Data needed to locate and identify the needed statement.
+	RelayParent common.Hash `scale:"1"`
+
+	// Hash of candidate that was used create the `CommitedCandidateRecept`.
+	CandidateHash CandidateHash `scale:"2"`
+}
+
+// Encode returns the SCALE encoding of the StatementFetchingRequest.
+func (s *StatementFetchingRequest) Encode() ([]byte, error) {
```

Comment:
Same feedback here, I dont think these are necessary are they?

Reply:
yup,
I have implemented the '[network.Message](https://github.com/ChainSafe/gossamer/blob/457fe39f7a12cb9eb04de5b6a315d38825247d68/dot/network/message.go#L30)' interface as 'StatementFetchingRequest' will be passed into [this](https://github.com/ChainSafe/gossamer/pull/3334/files#diff-5794e04e36068d98b7c6d2280c7e0b41fecca0b90bd6dd4576cd62b951ab587aR32) function as 'req'.